app/src/app/create/navtree/navtree.component.sass

```diff
@@ -29,3 +27,7 @@
   .placeholder.disabled-step
     border-bottom-style: dotted
     border-bottom-color: $disabled-frame-color
+  @media screen and (max-width: 414px)
```

Comment:
Please use a variable from responsive.sass instead of an hard-coded screen width limit. If there is no width in that file that suits you, you can add one, but keep in mind that there must be at most 4 breakpoints.